router/transformer/transformer.go

```diff
@@ -131,7 +131,7 @@ func (trans *handle) Transform(transformType string, transformMessage *types.Tra
 		req.Header.Set("X-Feature-Gzip-Support", "?1")
 		// Header to let transformer know that the client understands event filter code
 		req.Header.Set("X-Feature-Filter-Code", "?1")
-
+		req = req.WithContext(context.WithValue(req.Context(), "destination", &transformMessage.Data[0].Destination))
```

Comment:
1. Should we do this before `transformer.go` itself ?
2. Should we just send 1st destination object ? Is this ok ?!

Reply:
1. We can't do it before transformer.go as from the worker.go we are initializing it. We don't have access to the whole destination object in the worker. We can access the destinationDefinition only.
2. I need to check that. Is it possible events from multiple destinations are sent to the transformer?